src/main/scala/org/broadinstitute/dsde/firecloud/service/PerRequest.scala

```diff
@@ -63,6 +63,7 @@ trait PerRequest extends Actor {
     case _:HttpHeaders.Server => true
     case _:HttpHeaders.`Content-Type` => true
     case _:HttpHeaders.`Content-Length` => true
+    case _:HttpHeaders.`Transfer-Encoding` => true
```

Comment:
I found through testing that we were missed this definition in isAutomaticHeader.